pages/api/issues.js

```diff
@@ -1,7 +1,68 @@
-import issues from '../../data/issues.json'
-
-export default function handler(req, res) {
-    res.status(200).json(issues)
+const query = `
+query getIssues(
+  $labels: [String!], 
+  $owner: String!, 
+  $repo: String!
+) {
+  repository(owner: $owner, name: $repo) {
+    issues(last: 50, states: OPEN, labels: $labels) {
+      edges {
+        node {
+          title
+          url
+          labels(first: 5) {
+            edges {
+              node {
+                name
+                color
+              }
+            }
+          }
+          body
+          author {
+            ... on User {
+              avatarUrl
+              name
+              url
+            }
+          }
+          number
+          createdAt
+        }
+      }
+    }
   }
+}`;
+
+export default async function handler(req, res) {
+  const variables = {
```

Comment:
Why do we declare variables here but then pass req.query to the request body on line 61?